src/CloudModelGenerator/appSettings.json

```diff
@@ -0,0 +1,25 @@
+// This settings file will handle any default flags to pass to the application
+// Settings that are not set in this settings file will need to be passed via the command-line arguments
+{
```

Comment:
i'd simplify the format to just 
```
{
// Kentico Cloud Project ID
            "projectId" : "", 

            // Namespace name of the generated classes
            "namespace" : null, 

            // Output directory for the generated files
            "outputdir" : null, 

            // Optionally add suffix to the generated files
            "filenameSuffix" : null, 

            // Indicates whether the CustomTypeProvider class should be generated
            "withTypeProvder" : null, 
            
             // Indicates whether the classes should be generated with types that represent structured data model
            "structuredModel" : null 
}
```

it will make querying more simple as well